crates/wit/src/parsed_type/foreign_mod_prolog.rs

```diff
@@ -76,11 +77,11 @@ impl ForeignModPrologGlueCodeGenerator for Vec<(String, ParsedType)> {
                 arg_names.push(arg_ident.clone());
 
                 match ty {
-                    ParsedType::Utf8String => {
+                    ParsedType::Utf8String(PassingStyle::ByValue) => {
```

Comment:
`&str` isn't supported?

Reply:
Supported, but they don't need to be explicitly deleted.